### DISCUSSION TITLE
Release 1.11.0

### DISCUSSION THREAD

**Discussion by annube** [royhills/arp-scan#164] [General]
Hi,

some time has passed since 1.10 and quiet some cool features are waiting to be made public. Is a new release planned in the near future?

Cheers

**Comment by royhills** [comment]
Yes, I think it's time to plan for the 1.11.0 release. 1.10.0 has been out for almost a year now without any open bugs, and there's a few new features in the 1.10.1-git branch.

Thanks for opening this discussion.  I'll add progress updates here.

**Comment by ingoratsdorf** [comment]
Interesting. It's now more than 2 years after the last release yet work has been continuing.
Most packages out there for alpine and debian are ages old and I think still based on the 1.10 release?

Any reason for no new releases?

**Comment by lotron-boop** [comment]
I'm loving using arp-scan, however I'm having problems using custom mac mappings.

I run:
"arp-scan --localnet -m /usr/share/arp-scan/mac-vendor.txt -v"

and it keeps tossing a 
"WARNING: Could not parse oui: bc2411 Proxmox Virtual Interface"

I've read the docs, still no go, any hints?

**Reply by lotron-boop** [reply]
Nevermind!  The file is supposed to be tab delimited, now I've got it :-)
